packages/botfuel-dialog/src/adapters/shell-adapter.js

```diff
@@ -50,10 +52,47 @@ class ShellAdapter extends Adapter {
     });
   }
 
+  /* eslint-disable no-console */
   /** @inheritDoc */
   async sendMessage(botMessage) {
-    // eslint-disable-next-line no-console
-    console.log(chalk.hex('#16a085')(`${DELIMITER}${botMessage.payload.value}`));
+    if (botMessage.type === 'actions') {
+      await this.sendActionsMessage(botMessage);
+    } else {
+      console.log(chalk.hex(this.botTextColor)(`${DELIMITER}${botMessage.payload.value}`));
+    }
+  }
+
+  /**
+   * Send actions message
+   * @param {Object} actionsMessage actionMessage
+   * @returns {Promise.<void>}
+   */
+  async sendActionsMessage(actionsMessage) {
+    const actions = actionsMessage.payload.value;
+    let str = 'Please select:\n';
+    actions.forEach((action, index) => {
+      str += `${index + 1}: (${action.type}) ${action.text}\n`;
+    });
+
+    this.rl.question(chalk.hex(this.botTextColor)(str), async (answer) => {
```

Comment:
Are you sure the answer does not trigger an event `line`?
(an event listener is registered `this.rl.on('line', async (input) => {...`)

Reply:
I tested and it worked fine as in the image in the PR description